src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentPlanCard/FreePlanCard/FreePlanCard.test.jsx

```diff
@@ -291,7 +291,7 @@ describe('FreePlanCard', () => {
         wrapper,
       })
 
-      const link = await screen.findByRole('link', { name: /Manage plan/ })
+      const link = await screen.findByRole('link', { name: /Upgrade/ })
```

Comment:
do we know what caused this test to change?

Reply:
Not sure how it used to work, free plan should never have "manage plan" option